app/views/artworks/show.html.erb

```diff
@@ -1,12 +1,18 @@
 <div class="row">
-<%= link_to root_path do %>
-   <%= image_tag "back-arrow.svg", width: 44, height: 44, class: "back-arrow" %>
```

Comment:
I think Liv made the arrow into our brand colors, which is why it's an svg and rather than font awesome.